extra/src/main/scala/coursier/FallbackDependenciesRepository.scala

```diff
@@ -1,13 +1,11 @@
 package coursier
 
-import java.io.{ File, FileNotFoundException, IOException }
-import java.net.{ HttpURLConnection, URL, URLConnection }
-
+import java.io.{File, FileNotFoundException, IOException}
```

Comment:
Let's leave this file untouched since there's no functional change. If you would like to format the file, a separate PR would be better.